src/backend/InvenTree/company/migrations/0034_manufacturerpart.py

```diff
@@ -22,6 +22,7 @@ class Migration(migrations.Migration):
             ],
             options={
                 'unique_together': {('part', 'manufacturer', 'MPN')},
+                'verbose_name': 'Manufacturer Part',
```

Comment:
Q: Changing migrations like this after the fact is a bit unorthodox - is there a reason?

Reply:
This model never had a "verbose_name" defined - which means it doesn't get exposed to the translation layer. Adding it in requires a new migration, although the "verbose_name" does not touch the database. So this is a relatively safe "hack"